packages.txt

```diff
@@ -770,21 +766,23 @@ libsqlite3-dev
 libss2
 libssh2-1
 libssh-gcrypt-4
-libssl1.0.0
+libssl1.1
 libssl1.1
 libssl-dev
 libssl-doc
+libstdc++-10-dev
 libstdc++6
-libstdc++-6-dev
 libstdc++-7-dev
+libstdc++-8-dev
+libstdc++-9-dev
```

Comment:
Let's only have `libstdc++-9-dev`.

Reply:
There's `libstdc++-10-dev` above that. I'll retain just that, since it's the latest?